### PR TITLE
web: add the overview item popover

### PR DESCRIPTION
**Pull request by nicks**
Hello @hyu,

Please review the following commits I made in branch nicks/overviewitempopover:

bbeee3335a915bd71298eed88c7454a942924593 (2020-12-16 18:07:57 -0500)
web: add the overview item popover

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
Screenshot from 2020-12-16 18-11-54
Screenshot from 2020-12-16 18-10-35

**Comment by nicks** [comment]
In a follow up PR, I'll attach this to the main card

File: web/src/style-helpers.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export enum Color {
   white = "#ffffff",
 
   offWhite = "#eef1f1",
+  gray7 = "#CCDADE",
```
Comment (hyu) [inline web/src/style-helpers.ts:15]: Competing color naming schemes, commence!

File: web/src/OverviewItemView.tsx [hunk 2]
```diff
@@ -281,9 +291,164 @@ function buildTooltipText(status: ResourceStatus): string {
...
+      copyTextToClipboard(item.podId, () => {
+        setShowCopySuccess(true)
+
+        setTimeout(() => {
+          setShowCopySuccess(false)
+        }, 5000)
+      })
+    }
+
+    let icon = showCopySuccess ? (
```
Comment (hyu) [inline web/src/OverviewItemView.tsx:420]: Aw, nice! We should do more of this. There are still some janky "you copy it yourself 🙃" interactions scattered through Tilt...

File: web/src/OverviewItemView.stories.tsx [hunk 3]
```diff
@@ -111,3 +109,20 @@ export const OneItemLongName = () =>
...
+  return <OverviewItemDetails item={item} pathBuilder={pathBuilder} />
+}
+
+export const CompleteDetails = () => {
+  let item = new OverviewItem(oneResourceNoAlerts())
+  item.endpoints = [
+    { url: "http://localhost:4001" },
+    { url: "http://localhost:4002" },
+  ]
+  item.podId = "my-pod-deadbeef"
```
Comment (hyu) [inline web/src/OverviewItemView.stories.tsx:126]: 😱🐮

**Review by hyu — APPROVED** [review]

File: web/src/style-helpers.ts (hunk 1 above)
Comment (nicks) [reply web/src/style-helpers.ts:15]: hahaha, Surbhi and I had a convo on this yesterday...i think we should throw out the old UX first, then go back and reconcile all the color scales.

File: web/src/OverviewItemView.tsx (hunk 2 above)
Comment (nicks) [reply web/src/OverviewItemView.tsx:420]: i can't take credit for this, i copied it from this PR! https://github.com/tilt-dev/tilt/pull/3982

**Pull request merged**